converters/toZigbee.js

```diff
@@ -1168,6 +1168,13 @@ const converters = {
             return {state: {power_outage_memory: value}};
         },
     },
+    xiaomi_power: {
+        key: ['power'],
+        convertGet: async (entity, key, meta) => {
+            const endpoint = meta.device.getEndpoint(21);
```

Comment:
This line worries me a bit, it makes the converter specific to this device (as the other Xiaomi devices might not have this endpoint.

Can you try changing it to:

```js
const endpoint = meta.device.endpoints.find(e => e.supportsInputCluster("genAnalogInput"));
```

Reply:
Done